cloudsync/provider.py

```diff
@@ -120,6 +122,21 @@ def set_creds(self, creds):
         """Set credentials without connecting."""
         self._creds = creds
 
+    def set_root(self, root_path, root_oid):
```

Comment:
why do we need both?   seems like one or the other would do.

Reply:
we don't need both, changed the interface and impl in latest push (default both to None, which is also a valid configuration where a user-supplied translate function is used for mapping paths)